### PR TITLE
Remove mock data

### PR DESCRIPTION
**Pull request by larskarlitski**
Something simple for me to get to know the code a little.

The mock data is quite confusing when trying to see what's still missing.

We probably want empty states for "Changes" and the commit dropdown. If there are already designs for those, I can add them to this PR. 

@andreasn, @jgiardino Did I get everything?

### CODE REVIEW

**Comment by atodorov** [comment]
Can one of the admins verify this patch?

**Comment by atodorov** [comment]
Can one of the admins verify this patch?

**Comment by atodorov** [comment]
Can one of the admins verify this patch?

**Comment by atodorov** [comment]
Can one of the admins verify this patch?

**Comment by clumens** [comment]
There's also some lorem ipsum in components/Wizard/WizardView.js but I don't know what that file's used for, if anything.

File: pages/blueprint/index.js [hunk 1]
```diff
@@ -248,9 +200,9 @@ class BlueprintPage extends React.Component {
                       </button>
                     </dd>}
                   <dt>Install size</dt>
-                  <dd>2,678 KB</dd>
+                  <dd></dd>
                   <dt>Last modified date</dt>
-                  <dd>Thu,  9 Nov 2017</dd>
+                  <dd></dd>
```
Comment (jgiardino) [inline pages/blueprint/index.js]: Should we just remove this chunk of html altogether until we're actually pulling data in? At the moment, I don't think there's api support for showing install size either.
```
<dt>Install size</dt>
<dd></dd>
<dt>Last modified date</dt>
<dd></dd>
```

File: pages/blueprint/index.js [hunk 2]
```diff
@@ -285,15 +237,8 @@ class BlueprintPage extends React.Component {
                               data-toggle="dropdown"
                               aria-haspopup="true"
                               aria-expanded="false"
-                            >
-                              Commit 5<span className="caret" />
-                            </button>
+                            ></button>
```
Comment (jgiardino) [inline pages/blueprint/index.js]: Similar to previous comment, I think we could remove this <button> altogether until the functionality for this implemented.

File: pages/blueprint/index.js [hunk 3]
```diff
@@ -36,56 +36,8 @@ class BlueprintPage extends React.Component {
...
-        },
-        {
-          date_created: '1/17/17',
-          date_exported: '1/17/17',
-          user: 'Brian Johnson',
-          type: 'iso',
-          change: '2',
-          size: '1,234 KB',
-        },
-      ],
```
Comment (jgiardino) [inline pages/blueprint/index.js]: The UI associated with this data is hidden except for the [section header "Changes"](https://github.com/weldr/welder-web/pull/250/files#diff-a72936711756a1ad3bf1f32d6b45f83dR210). Should we just remove this chunk of UI, or conditionally display it when data is available?

image

**Comment by andreasn** [comment]
> Did I get everything?

screenshot from 2018-06-28 00-29-55

I _think_ install size here is hard-coded, so that's also probably good to get rid of.
Is that correct, @jgiardino ?

**Comment by andreasn** [comment]
There is also the images pane under Edit Blueprints.
screenshot from 2018-06-28 00-42-37

**Comment by jgiardino** [comment]
> There is also the images pane under Edit Blueprints.

The mock data for images is also being removed as part of #245, but I think it's fine to remove it as part of this PR.

> I think install size here is hard-coded, so that's also probably good to get rid of.

There are currently two places where metadata displays:
- [When clicking on a component name](https://github.com/weldr/welder-web/blob/master/components/ListView/ComponentDetailsView.js)
  - In this case, there is also an ["Errata" tab](https://github.com/weldr/welder-web/blob/master/components/ListView/ComponentDetailsView.js#L339) that can be removed
- [When expanding a component list item](https://github.com/weldr/welder-web/blob/master/components/ListView/ListItemComponents.js)

In these cases, the following component metadata can be removed:

- install size
- packager
- product family
- lifecycle
- support level
- architecture*

 *There is a [user story in Jira](https://projects.engineering.redhat.com/browse/COMPOSER-62) for adding info for "Architecture", but this is the only metadata value for which there's any api support.

**Comment by jgiardino** [comment]
> There's also some lorem ipsum in components/Wizard/WizardView.js but I don't know what that file's used for, if anything.

This is correct. I removed this, along with other non-functional or unused UI in #252.

I think between this PR, #252, #248, and #245, that all mock UI elements are covered.

**Comment by larskarlitski** [comment]
Pushed a new commit were I've also removed the additional things you suggested. Thanks @jgiardino and @andreasn!

**Comment by jgiardino** [comment]
Looks great. Thanks!

**Comment by jgiardino** [comment]
Oooh, there's one tiny lint error. You can check with `npm run lint`

**Comment by andreasn** [comment]
Works for me!
What's up with Selected Components? I keep getting just one component there, but I'm not sure where it comes from, or what function it serves. Is that being tracked anywhere?

**Comment by jgiardino** [comment]
> What's up with Selected Components? I keep getting just one component there, but I'm not sure where it comes from, or what function it serves.

Hmm, I'm not sure what's happening there. Do you happen to remember what actions you took that resulted in that? If you create a new blueprint, do you still see one selected component?

**Comment by andreasn** [comment]
> If there is an issue, I don't think it's part of the mock data, so we could track that separately from this PR.

Filed as #265

**Comment by larskarlitski** [comment]
Thanks for noticing the error, @jgiardino. Fixed the lint error, rebased and resolved conflicts.

**Comment by henrywang** [comment]
Please ignore unit test failure. It's not caused by this PR's code change.
@jgiardino, feel free to merge this PR if you don't have comment. Thanks.

**Comment by jgiardino** [comment]
Thanks for the review, @henrywang!

**Pull request merged**